### ISSUE TITLE
Changing file content refreshes the GLTF Viewer view, but always resets to Babylon.js

### ISSUE THREAD

**Issue by soraryu** [AnalyticalGraphicsInc/gltf-vscode#176]
**Expected**: changing file content refreshes the preview and keeps the previously selected engine selected.
**Actual**: engine is reset to "Babylon.js" on file change.

![191220-133644](https://user-images.githubusercontent.com/2693840/71255298-d1e27080-232d-11ea-8490-bceeb15629c5.gif)

**Comment by emackey** [comment]
Yes, it should probably remember whatever engine it was on before.

Note there is a setting for which engine is the "default" for glTF 2.0 models, in case you want to have ThreeJS (or Cesium) come up first.

Also I should add a setting to enable (by default) the auto-updating behavior, so folks can turn that off entirely if they prefer.

**Comment by soraryu** [comment]
Thanks for the response! Good to know there's a setting for the default.

However, at least my usecase here is preparing models so that they are working in different render engines, so I would like it to stay on whatever I had selected. Usually when I find an issue with an engine, I update the files and want to check what that change did in the same engine.
